src/chemcalculator/chemcalculator.py

```diff
@@ -82,7 +82,7 @@ def percent_mass(compound, element):
     88.79
     
     >>> percent_mass("H2O", "H")
-    11.19
+    5.6
```

Comment:
We'll need to brainstorm something because as calculate_mass is currently written, sending 'H' will return 1.0007 so for 'H20', it would actually return 2.8 and not 5.6/

Reply:
I'm confused why it would return 2.8. I used 1.0007 to calculate and it should return 5.553%

Reply:
I think it would return 5.553% by using 1.007 and the function percent_mass() As it's H2 in the H2O as opposed to H